bot/exts/evergreen/coinflip.py

```diff
@@ -0,0 +1,57 @@
+import random
+from typing import Optional, Tuple
+
+from discord.ext import commands
+
+from bot.bot import Bot
+
+
+LEMON_HYPERPLEASED = "<:lemon_hyperpleased:754441879822663811>"
+LEMON_PENSIVE = "<:lemon_pensive:754441880246419486>"
```

Comment:
These emojis should probably be in the constants file, not here, in case they are wanted elsewhere/people want to change them locally

Reply:
Have moved these into `constants.py` under the `Emojis` class :+1: